### DISCUSSION TITLE
Transform "Reduce", possibility of adding a "Reduce visible"?

### DISCUSSION THREAD

**Discussion by TryTryAgain** [grafana/grafana#62888] [Dashboards, Panels & Visualizations]
This may be related to other long-awaited features to hide/reduce only what's visible (I know I'm following the one to omit empty repeated panels for example, and I understand there are larger underlying challenges and considerations; I'd have to assume the same applies for this request as well), but more specifically to the Transform "Reduce" I'm wondering about the likelihood something like "Reduce visible only" could be introduced... 

The Reduce Transform is very useful but doesn't make sense when the data is static and you go and hide particular metrics in the dashboard but the total remains the same. It would be very useful to also have a "Reduce visible" that would dynamically update to reflect only what is visible in the graph.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82933. Feel free to continue the discussion around this there. Thank you!

**Reply by TryTryAgain** [reply]
Thanks for your help, hope it gets some attention as an issue :D ... at least it didn't get closed outright.
